spec/integration/google_tag_manager_integration_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'support/capybara_app_helper'
+
+RSpec.describe "Google Tag Manager Integration" do
+  before do
+    setup_app(action: :google_tag_manager) do |tracker|
+      tracker.handler :google_tag_manager, { tracker: 'GTM-ABCDEF' }
+    end
+    visit '/'
+  end
+
+  subject { page }
+
+  it "embeds the script tag with tracking event from the controller action" do
+    expect(page.find("head")).to have_content('GTM-ABCDEF')
```

Comment:
`noscript` seems to be the issue, anyways google recommends to put the snippit in the body.

also see: http://www.lunametrics.com/blog/2014/12/12/google-tag-manager-snippet-placement/